llvm/lib/DebugInfo/Symbolize/SymbolizableObjectFile.cpp

```diff
@@ -265,28 +265,39 @@ bool SymbolizableObjectFile::shouldOverrideWithSymbolTable(
          isa<DWARFContext>(DebugInfoContext.get());
 }
 
-DILineInfo
-SymbolizableObjectFile::symbolizeCode(object::SectionedAddress ModuleOffset,
+DILineInfo SymbolizableObjectFile::symbolizeCode(object::SectionedAddress ModuleOffset,
                                       DILineInfoSpecifier LineInfoSpecifier,
                                       bool UseSymbolTable) const {
-  if (ModuleOffset.SectionIndex == object::SectionedAddress::UndefSection)
-    ModuleOffset.SectionIndex =
-        getModuleSectionIndexForAddress(ModuleOffset.Address);
+  if (ModuleOffset.SectionIndex == object::SectionedAddress::UndefSection) {
+    ModuleOffset.SectionIndex = getModuleSectionIndexForAddress(ModuleOffset.Address);
+  }
   DILineInfo LineInfo =
       DebugInfoContext->getLineInfoForAddress(ModuleOffset, LineInfoSpecifier);
 
-  // Override function name from symbol table if necessary.
-  if (shouldOverrideWithSymbolTable(LineInfoSpecifier.FNKind, UseSymbolTable)) {
-    std::string FunctionName, FileName;
-    uint64_t Start, Size;
-    if (getNameFromSymbolTable(ModuleOffset.Address, FunctionName, Start, Size,
-                               FileName)) {
-      LineInfo.FunctionName = FunctionName;
-      LineInfo.StartAddress = Start;
-      if (LineInfo.FileName == DILineInfo::BadString && !FileName.empty())
-        LineInfo.FileName = FileName;
+  // HACK: Always provide symbol table function name
+  std::string FunctionName, FileName;
+  uint64_t Start, Size;
+  if (getNameFromSymbolTable(ModuleOffset.Address, FunctionName, Start, Size,
```

Comment:
Should `FunctionName` be `SymbolTableFunctionName` to align with the [original patch](https://github.com/bugsnag/llvm/commit/cf822825b7bb57179da1e835a4b50f19d4e83c28#diff-2ef25f78b9a31e543e51437cd178610b19b22662f1de12196435a839c4e9bfa6R246-R248)?

Reply:
Also, [this patch](https://github.com/bugsnag/llvm/commit/cf822825b7bb57179da1e835a4b50f19d4e83c28#diff-2ef25f78b9a31e543e51437cd178610b19b22662f1de12196435a839c4e9bfa6R265-R271) doesn't appear to be applied